themes/nwinkler_random_colors/nwinkler_random_colors.theme.bash

```diff
@@ -15,7 +15,7 @@
 # The exit code functionality currently doesn't work if you are using the 'fasd' plugin,
 # since 'fasd' is messing with the $PROMPT_COMMAND
 
-RANDOM_COLOR_FILE=$HOME/.nwinkler_random_colors
+RANDOM_COLOR_FILE=${NWLINKLER_RANDOM_COLOR_FILE:-$HOME/.nwinkler_random_colors}
```

Comment:
While I find it rather flattering that this theme keeps getting used, it looks like you have a typo in the new variable name. It should be `NWINKLER`, not `NWLINKLER`. Can you please change that?